notebooks/model_introspection/principal_filter_analysis.ipynb

```diff
@@ -110,7 +110,7 @@
    },
    "source": [
     "### 1.c Find Convolution layers\n",
-    "Similarly to the prior cell, here all layers are parsed in search of `Conv2D` layers. This is the set of layers that to analyze using PFA. If the list of names of the layers to analyze is already known, it can be passed directly to the loaded model (see step 1.f).\n",
+    "Similar to the prior cell, here all layers are parsed in search of `Conv2D` layers. This is the set of layers that will be used to analyze with PFA. If the list of names of the layers to analyze is already known, it can be passed directly to the loaded model (see step 1.e).\n",
```

Comment:
```suggestion
    "Similar to the prior cell, here all layers are parsed in search of `Conv2D` layers. This is the set of layers that will be analyzed with PFA. If the list of names of the layers to analyze is already known, it can be passed directly to the loaded model (see step 1.e).\n",
```